### DISCUSSION TITLE
Interactable webpage?

### DISCUSSION THREAD

**Discussion by cheetozzc** [mozilla/hubs#5645] [General Questions & Discussion] [answered]
Hi, I am new to hubs! I wonder if it's possible to have an Interactable webpage as an element rather than a screenshot of the page? or we already can do it ? Thanks!

**Comment by matthewbcool** [comment, marked as answer]
Hello! It is possible to share your screen in Hubs and then use [media frames](https://hubs.mozilla.com/docs/spoke-adding-scene-content.html#media-frames) to pre-define the spaces you would like it to occupy. 

iframes and virtual machines can make it possible to interact with shared external web content in-room are possible solutions some folks have built with Hubs Cloud but are not built-in features currently on hubs.mozilla.com. 

Welcome to join us in [discord](https://discord.gg/sBMqSjCndj) if you would like to chat more and meet with the team directly at the community meetup Friday's. Details in our discord calendar.
